devel/gaa/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem               1.0
+PortGroup                github 1.0
+
+github.setup             cooljeanius gaa 1.6.6_1 v
+revision                 3
+platforms                darwin
+categories               devel
+maintainers              gwmail.gwu.edu:egall openmaintainer
+description              ${name} Argument Analyser
+license                  GPL-2+
+long_description         ${name} simplifies the life of the programmer: \
+                         you do NOT have to worry about the arguments \
+                         given to it! A short text written in GAA \
+                         language generates C-code which analyses \
+                         the arguments and creates your program's help.
+
+homepage                 http://${name}.sf.net
+
+fetch.type               git
+git.url                  https://github.com/cooljeanius/${name}.git
+
+depends_build-append     port:bison \
+                         port:flex \
+                         port:gmake \
+                         bin:makeinfo:texinfo
+
+depends_lib-append       port:yydecode
+depends_lib-delete       port:gobject-introspection
+
+configure.ccache         no
+
+configure.args-append    --with-bison-bin=${prefix}/bin/bison
+
+build.type               gnu
+build.cmd                ${prefix}/bin/gmake
+
+variant autoreconf description {Regenerates configure script before \
+                                building. Also pulls in extra \
+                                dependencies.} {
+    depends_build-append port:gawk \
+                         port:grep \
+                         port:autoconf-archive
+    use_autoreconf       yes
+    autoreconf.args      -fvi -Wall
+    configure.args-append --disable-silent-rules
+}
+
+variant docs description {Generate additional documentation by \
+                          using latex2html} {
+    depends_build-append port:latex2html \
+                         port:texlive-latex
+    depends_lib-delete   port:poppler
+    use_parallel_build   no
+    post-build {
+        system -W ${worksrcpath}/doc "${prefix}/bin/latex2html *.tex"
+    }
+    post-destroot {
+        foreach docdir {gaa ref tut} {
+            xinstall -d ${destroot}${prefix}/share/doc/${name}/${docdir}
+            xinstall -m 644 {*}[glob ${worksrcpath}/doc/${docdir}/*] ${destroot}${prefix}/share/doc/${name}/${docdir}
+        }
+    }
+}
+
+livecheck.type           none
```

Comment:
The livecheck should work automatically as it is implicitly set by `github.setup`.

Reply:
ah, the livecheck reminded me that I actually have a new tag to which I can update; let me do that as well...

Reply:
...actually never mind; updating to my latest tag breaks the documentation build, so I'll just stick with the current tag for now...

Reply:
OK, I removed the livecheck line in [3e0470f](https://github.com/macports/macports-ports/pull/20972/commits/3e0470fc5ad30184fec83c23722be7b056bbd219).
(Performing the actual update suggested by the livecheck will remain a task for another day...)